### PR TITLE
chore: add general PR template

### LINKED ISSUE #4: Add general PR template
We see a lot of PRs with little to non description about the introduces checks. Also, the copyright and license headers are commonly forgotten and DEPENDENCIES are out of date.

We should provide a general PR template with information or checks, so as soon as someone opens a PR, a reminder of the recurring activities is present

### PR DESCRIPTION
**Pull request by SebastianBezold**
## Description 

This PR introduces an org wide pull request template with a basic structure, that contributors can follow.
Also included is a list of checks, that contributors should complete, before asking committers for review. 
A list with links to further reading and project guidelines is included at the end.

fixes #4

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+about: 'template for PRs that ensures basic information is provided'
+title: '(feat|fix|chore|doc): _description of introduced change_'
+---
+
+## Description
+
+Thanks for opening this contribution!
+_What does this PR introduce? Does it fix a bug? Does it add a new feature? Is it enhancing documentation?_
+
+Fixes # (issue)
```
Comment (FaGru3n) [inline .github/pull_request_template.md]: i would add `(https://github.com/eclipse-tractusx/<product-repo>/issues/<number-of-the-issue>)`
to make it clear to open issues for ETX

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Pre-review checks
+
+Please ensure to do as many of the following checks as possible, before asking for committer review:
+
+- [ ] DEPENDENCIES are up to date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
+- [ ] Copyright and license header are present on all affected files
+
```
Comment (FaGru3n) [inline .github/pull_request_template.md:20]: we could also add here the committer guidelines from Eclipse (https://www.eclipse.org/legal/committerguidelines.php)

**Review by FaGru3n — CHANGES_REQUESTED** [review]
Looks good for the Commiter Round... just small addition from myside

File: .github/pull_request_template.md (hunk 1 above)
Comment (SebastianBezold) [reply .github/pull_request_template.md]: Hmm. If you open a PR and add an issue via '#', you are anyways tied to issues in the repo you open the PR on

Comment (FaGru3n) [reply .github/pull_request_template.md]: legit

**Review by Siegfriedk — COMMENTED** [review]
Can we also add a hint on how to contribute to this template?

**Comment by Essenbreis** [comment]
Is there a possibility to add security checks in this template? We could elaborate together in a meeting how this could look like @the-tatanka - What would be your ideas?

**Comment by Siegfriedk** [comment]
> 

@Essenbreis we can! We should! :) unfortunate our security meeting was skipped due to a few meetings but it will happen this week. So perhaps a topic for that.

**Comment by scherersebastian** [comment]
> Is there a possibility to add security checks in this template? We could elaborate together in a meeting how this could look like @the-tatanka - What would be your ideas?

if we go full open source, that's a good option.

**Review by FaGru3n — APPROVED** [review]
LGTM 👍🏼 

@Essenbreis @the-tatanka  maybe we should add this as a separat issue or task if you want

**Comment by almadigabor** [comment]
What's the status here? Can we merge and close? Adding security checks can be discussed in a separate new issue.

**Comment by Essenbreis** [comment]
Hi Gabor, I agree security checks can be discussed in a separate new issue, We first have to align internally our security strategy. You can merge and close.

**Comment by almadigabor** [comment]
Great, thanks for the heads up @Essenbreis! :) As soon as you're aligned internally, feel free to open an issue [on our board](https://github.com/eclipse-tractusx/sig-infra/issues/new/choose),

**Pull request merged**